gobblin-iceberg/src/test/java/org/apache/gobblin/iceberg/writer/HiveMetadataWriterTest.java

```diff
@@ -128,7 +128,6 @@ public void clean() throws Exception {
   @BeforeSuite
   public void setUp() throws Exception {
     Class.forName("org.apache.derby.jdbc.EmbeddedDriver").newInstance();
-    startMetastore();
```

Comment:
are we not starting the local Metastore for tests now?

Reply:
the local Metastore is now started by the parent class: `HiveMetastoreTest` with the upgrade, since they now added `@BeforeClass` annotation for `startMetastore()`. Here's the [Junit api doc](https://junit.org/junit4/javadoc/latest/org/junit/BeforeClass.html) for the same. Thus, we do not need to start again for our unit tests otherwise we would run into this issue:`AlreadyExistsException(message:Database hivedb already exists)`. Alternative would be to try starting again on our side and catch the above exception and handle it.